### PR TITLE
Add `ignore_initial_volumes` param to `ConfoundsCorrelationPlot`

### PR DESCRIPTION
**Pull request by tsalo**
Related to https://github.com/nipreps/niworkflows/pull/843.

Changes proposed:

- Add an `ignore_initial_volumes` parameter to the `ConfoundsCorrelationPlot` interface.
- Add an `ignore_initial_volumes` parameter to the `confounds_correlation_plot` function.

### CODE REVIEW

**Comment by effigies** [comment]
How tough do you think it would be to add a unit test?

**Comment by tsalo** [comment]
I think I can just copy over the unit tests from niworkflows, so probably not too hard. I'll try to do it by EOD.

File: nireports/tests/test_interfaces.py [hunk 1]
```diff
@@ -21,3 +21,45 @@
 #     https://www.nipreps.org/community/licensing/
 #
 """Tests plotting interfaces."""
+import os
+from shutil import copy
+from nireports.interfaces.nuisance import CompCorVariancePlot, ConfoundsCorrelationPlot
```
Comment (effigies) [inline nireports/tests/test_interfaces.py:26]:
```suggestion

import pytest

from nireports.interfaces.nuisance import CompCorVariancePlot, ConfoundsCorrelationPlot
```

File: nireports/tests/test_interfaces.py [hunk 2]
```diff
@@ -21,3 +21,45 @@
...
+def test_ConfoundsCorrelationPlotColumns(datadir):
+    """confounds correlation report test"""
+    confounds_file = os.path.join(datadir, "confounds_test.tsv")
+    cc_rpt = ConfoundsCorrelationPlot(
+        confounds_file=confounds_file,
+        reference_column="a",
+        columns=["b", "d", "f"],
+        ignore_initial_volumes=0,
+    )
+    _smoke_test_report(cc_rpt, "confounds_correlation_cols.svg")
```
Comment (effigies) [inline nireports/tests/test_interfaces.py:65]:
```suggestion
@pytest.mark.parametrize(ignore_initial_volumes, (0, 1))
def test_ConfoundsCorrelationPlot(datadir):
    """confounds correlation report test"""
    confounds_file = os.path.join(datadir, "confounds_test.tsv")
    cc_rpt = ConfoundsCorrelationPlot(
        confounds_file=confounds_file,
        reference_column="a",
        ignore_initial_volumes=ignore_initial_volumes,
    )
    _smoke_test_report(cc_rpt, f"confounds_correlation_{ignore_initial_volumes}.svg")
```

File: nireports/tests/test_interfaces.py [hunk 3]
```diff
@@ -21,3 +21,37 @@
...
+
+def test_CompCorVariancePlot(datadir):
+    """CompCor variance report test"""
+    metadata_file = os.path.join(datadir, "confounds_metadata_test.tsv")
+    cc_rpt = CompCorVariancePlot(metadata_files=[metadata_file], metadata_sources=["aCompCor"])
+    _smoke_test_report(cc_rpt, "compcor_variance.svg")
+
+
+@pytest.mark.parametrize(ignore_initial_volumes, (0, 1))
+def test_ConfoundsCorrelationPlot(datadir):
```
Comment (effigies) [inline nireports/tests/test_interfaces.py:49]:
```suggestion
@pytest.mark.parametrize('ignore_initial_volumes', (0, 1))
def test_ConfoundsCorrelationPlot(datadir, ignore_initial_volumes):
```

**Pull request merged**